helm/robusta/values.yaml

```diff
@@ -74,6 +74,7 @@ lightActions:
 
 # install prometheus, alert-manager, and grafana along with Robusta?
 enablePrometheusStack: false
+enableManagedPrometheusAlerts: false
```

Comment:
I think this should be renamed to `enabledManagedConfiguration`
A single value to all types of managed configurations